Clinical-Genomics/cg
# Allow non saved fields for existing samples to be optional

This PR fixes the problem to submit orders with families that you have added existing samples to.

How to test:
0. install on stage (add instructions on how to as needed)
1. run test in https://github.com/Clinical-Genomics/cgweb/pull/98

Expected outcome:
Success
